docs/getting_started/0_scripts_quickstart/5_sql_quickstart/index.mdx

```diff
@@ -0,0 +1,149 @@
+[Windmill Hub][hub-resources].
+
+:::
+
+Head to the <a href="https://app.windmill.dev/resources" rel="nofollow">Resources</a> page in the Windmill app, click on
+"Add a resource/API" in the top right corner and select the `PostgreSQL` or `MySQL` type.
+
+![Select Resource Type](../../../assets/integrations/psql-1-resources.png)
+
+Fill out the form with the information of your PostgreSQL/MySQL instance.
+
+:::tip
+
+For testing purposes, you can use the sample PostgreSQL Resource provided to
+every user. It is available under the path `f/examples/demodb`.
+
+:::
+
+![Paste in Resource Values](../../../assets/integrations/psql-2-postgres-rt.png)
+
+## Create Script
+
+Next, let's create a script that will use the newly created Resource. Head on to
+the <a href="https://app.windmill.dev/" rel="nofollow">Home</a> page and click on the "+Script" button. Name the Script, give it a summary, "Query a PostgreSQL Database" for
+example and select your prefered language, "PostgreSQL" or "MySQL".
+
+![Script creation first step](../../../assets/integrations/psql-3-script-creation.png)
+
+You can also give more details to your script, in the [metadata section](../../../script_editor/metadata.mdx), you can also get back to that later at any point.
+
+### PostgreSQL
+
+Arguments need to be passed in the given format:
+
+```sql
+-- $1 name1 = default arg
+-- $2 name2
+INSERT INTO demo VALUES ($1::TEXT, $2::INT) RETURNING *
+```
+
+"name1", "name2" being the names of the arguments, and "default arg" the optional default value.
+
+You can then write your prepared statement.
+
+![Postgresql statement](./postgresql_statement.png)
+
+### MySQL
+
+Arguments need to be passed in the given format:
+
+```sql
+-- ? name1 (text) = default arg
+-- ? name2 (int)
+INSERT INTO demo VALUES (?, ?)
+```
+
+"name1", "name2" being the names of the arguments, and "default arg" the optional default value.
+
+You can then write your prepared statement.
+
+![Mysql statement](./mysql_statement.png)
+
+### Raw Queries
+
+A more convenient but less secure option is to accept raw queries as input.
+
+This method will imply going through a [Typescript script](../1_typescript_quickstart/index.mdx): [Execute Query and return results](https://hub.windmill.dev/scripts/postgresql/1294/execute-query-and-return-results-postgresql).
+
+```typescript
+import { pgClient, type Sql } from 'https://deno.land/x/windmill@v1.88.1/mod.ts';
+
+type Postgresql = {
+	host: string;
+	port: number;
+	user: string;
+	dbname: string;
+	sslmode: string;
+	password: string;
+};
+
+export async function main(db: Postgresql, query: Sql = 'SELECT * FROM demo;') {
+	if (!query) {
+		throw Error('Query must not be empty.');
+	}
+	const { rows } = await pgClient(db).queryObject(query);
+	return rows;
+}
+```
+
+This will allow you to execute all commands from one script. But that's also the vulnerability of it as it opens the door to harmful commands.
```

Comment:
To make it more secure, do not take the query directly as input but its parameters and then sanitize it properly so SQL injections are mitigated unless you are in a trusted environment.